mail_drop_target/models/res_config_settings.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019-20 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    disable_notify_mail_drop_target = fields.Boolean(
```

Comment:
MAybe you can use `config_parameter='mail_drop_target.disable_notify'` in order to avoid the functions

Reply:
@JoanSForgeFlow what about this?

Reply:
Done!